src/Integration.UnitTests/Telemetry/TelemetryManagerTests.cs

```diff
@@ -213,6 +182,11 @@ public void WhenTryUploadDataTimerElapsedAndNewDay_ChangeLastUploadAndSaveAndSen
             telemetryData.ShowHotspot.NumberOfRequests.Should().Be(0); // should have cleared the counter
             telemetryData.TaintVulnerabilities.NumberOfIssuesInvestigatedRemotely.Should().Be(0); // should have cleared the counter
             telemetryData.TaintVulnerabilities.NumberOfIssuesInvestigatedLocally.Should().Be(0); // should have cleared the counter
+            telemetryData.ServerNotifications.IsDisabled.Should().BeFalse(); // should have cleared the value
```

Comment:
Ideally, TelemetryDataRepository would have a "ClearDailyCounters" method (or something) that the manager could call, without knowing the internal structure. But I left this as-is for now, we need to think first about the overall design.